### PR TITLE
Pass request_context to resolvers as a context value

### PR DESCRIPTION
**Pull request by ciscorn**
Request objects should be accessible via `info.context`, like sanic-graphql, aiohttp-graphql, etc.

### CODE REVIEW

File: graphql_ws/base.py [hunk 1]
```diff
@@ -59,22 +59,22 @@ def __init__(self, schema, keep_alive=True):
...
     def get_graphql_params(self, connection_context, payload):
-        return {
+        params = {
             'request_string': payload.get('query'),
             'variable_values': payload.get('variables'),
             'operation_name': payload.get('operationName'),
             'context_value': payload.get('context'),
         }
-
-    def build_message(self, id, op_type, payload):
```
Comment (ciscorn) [inline graphql_ws/base.py]: `build_message` is no longer used

**Comment by subwindow** [comment]
Is there any chance of this getting merged? This is blocking me, as there's no way to pass the request to a resolver.

**Comment by messa** [comment]
@subwindow 

> Is there any chance of this getting merged? This is blocking me, as there's no way to pass the request to a resolver.

@cito is working on graphql-ws-next, hopefully will be published soon.

In the meantime you can "monkeypatch" this library - see https://github.com/graphql-python/graphql-ws/issues/7#issuecomment-412771612 or [the same thing in my demo project](https://github.com/messa/aiohttp-nextjs-graphql-demo-forum/blob/master/backend/forum_backend/app.py#L24).

**Comment by subwindow** [comment]
Awesome, that worked. Thanks!

**Pull request closed without merging**